great_expectations/rule_based_profiler/helpers/simple_semantic_type_filter.py

```diff
@@ -68,19 +70,20 @@ def parse_semantic_domain_type_argument(
                     SemanticDomainTypes(semantic_type)
                     for semantic_type in semantic_types
                 ]
-            elif all(
+
+            if all(
                 [
                     isinstance(semantic_type, SemanticDomainTypes)
                     for semantic_type in semantic_types
                 ]
```

Comment:
Nit - we don't need to create a list here. `all` works with any iterator so the extra memory is unnecessary

Reply:
@cdkini -- Incorporated -- thanks.